manifests/test/patch.yaml

```diff
@@ -4,3 +4,4 @@
 - op: add
   path: /spec/template/spec/containers/0/imagePullPolicy
   value: Never
+imagePullPolicy: Always
```

Comment:
Why do you want to make this modification? Is there a related issue or description?

Reply:
Hi,

imagePullPolicy: Always(Mandatory parameter to update in manifest/test/patch.yml)
to run this command:kubectl top pods-which gives CPU usage and its important command,I did modification as per the document and raised pull request. Please approve and merge it.

Reply:
Will you use this file? 
I don't understand how this file affects the command `kubectl top pods`

Reply:
In the manifests file of release, this configuration is not added, such as `https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.5.2/components.yaml`

Reply:
Agree. 
@rameshchelliah1989, We will close this pr. thank you.
/close